src/components/Footer/Footer.js

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+
```

Comment:
Consider removing redundant empty lines.
The rule is to have only one empty line that separates blocks of grouped code.